core/src/test/java/org/apache/calcite/test/UdfTest.java

```diff
@@ -219,6 +225,35 @@
     withUdf().query(sql).returns(expected);
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-3195">[CALCITE-3195]
+   * Handle UDF that throws checked exceptions in enumerable code generator</a>. */
```

Comment:
I suggest rewording to "Handle **a** UDF that throws in **the Enumerable** code generator"